frontend/src/components/VContentReport/VContentReportButton.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <VButton
-    variant="plain"
-    class="report-button font-semibold text-dark-charcoal-70 hover:underline"
+    variant="transparent-gray"
+    size="medium"
+    has-icon-end
+    class="description-bold text-dark-charcoal-70 !pe-0 md:text-base"
```

Comment:
I didn't communicate my intention correctly about reducing the right space. I was referring to the layout rather than the button itself. But after testing it, it looks better bringing the right space back and remove the `!pe-0` value

Reply:
I removed the 'pe-0', thank you!